src/app/routes.jsx

```diff
@@ -11,7 +11,7 @@ export const Routes = () => (
     <ProtectedRoute
       path="/catalogs/:catalogId"
       check={({ catalogId }) =>
-        authService.hasOrganizationReadPermission(catalogId)
+        authService.hasSystemAdminPermission() || authService.hasOrganizationReadPermission(catalogId)
```

Comment:
Her ser det ut som du sjekker både ut fra det som kan være en rolle eller en permission. Vi har vel definert bare to permissions, read og write. Kan vurdere å aggregere permissions til en funksjon, f.eks.hasAdminRole, dvs. har både read og write permission. Hvis vi senere tenker oss å legge til flere permissions, så legger vi bare det til i rolle-funksjon.

Reply:
Det er visse som ikke skal vises til sys-admin brukeren uten at brukeren i tillegg har en faktisk org-tilgang. Hvis vi sier at sys-admin alltid har org-write så blir det vankselig å skille